src/imagery/i.eodag/i.eodag.html

```diff
@@ -57,18 +57,24 @@ <h3>WARNING: I.EODAG IS UNDER DEVELOPMENT. THIS IS AN EXPERIMENTAL VERSION.</h3>
 
 <h2>EXAMPLES</h2>
 
-Search and list the available Sentinel 2 scenes in the Copernicus Data Space Ecosystem:
+Search and list the available Sentinel 2 scenes in the Copernicus Data Space Ecosystem, using a Vector Map as an AOI:
 
 <div class="code"><pre>
 i.eodag -l start=2022-05-25 end=2022-06-01 \
-    dataset=S2_MSI_L2A provider=cop_dataspace
+    map=vector_map_sample dataset=S2_MSI_L2A provider=cop_dataspace
```

Comment:
```suggestion
    map=durham dataset=S2_MSI_L2A provider=cop_dataspace
```

Reply:
We want examples to be reproducible as much as possible, this is why we usually use NC sample dataset